cmd/namespace/use.go

```diff
@@ -155,7 +155,9 @@ func getNamespacesSelection(ctx context.Context) ([]utils.SelectorItem, error) {
 
 func askForOktetoNamespace() string {
 	var namespace string
-	oktetoLog.Question("Enter the namespace you want to use: ")
+	if err := oktetoLog.Question("Enter the namespace you want to use: "); err != nil {
```

Comment:
Same with this one. If we can't throw a question we should return the proper error

Reply:
Should I go for `oktetoLog.Errorf()` here since the func doesn't return an error?

Reply:
Let's modify the function signature to `func askForOktetoNamespace() (string, error)`